src/model/openthaigpt_pretraining_model/llama_thai_tokenizer/merge_tokenizer.py

```diff
@@ -0,0 +1,50 @@
+from transformers import LlamaTokenizer
+from sentencepiece import sentencepiece_model_pb2 as sp_pb2_model
+import sentencepiece as spm
+import os
+
+LLAMA_TOKENIZER_DIR = (
+    "/root/openthaigpt-pretraining/src/model/openthaigpt_pretraining_model/"
```

Comment:
I think this line can not run in every environment. I suggest you to declare `FILE_DIR` constants. For example
```python
import os
FILE_DIR = os.path.dirname(__file__)
LLAMA_TOKENIZER_DIR = f"{FILE_DIR}/path/to/file"
```

Reply:
done